### PR TITLE
End audio devicer selection menu pain

### PR DESCRIPTION
**Pull request by howard-stearns**
Disconnect menuItemEvent before setIsOptionChecked, and then reconnect.

### CODE REVIEW

**Comment by howard-stearns** [comment]
This gives us the option of an emergency fix to https://app.asana.com/0/26225263936266/106399297514513
while giving me time to look more into a better/more-comprehensive fix.

File: examples/selectAudioDevice.js [hunk 1]
```diff
@@ -120,21 +120,25 @@ function menuItemEvent(menuItem) {
         if (menuItem.endsWith(" for Output")) {
             var selectedDevice = menuItem.trimStartsWith("Use ").trimEndsWith(" for Output");
             print("output audio selection..." + selectedDevice);
+	    Menu.menuItemEvent.disconnect(menuItemEvent);
```
Comment (ZappoMan) [inline examples/selectAudioDevice.js]: hmmm this fix seems kinda janky... the issue is that we're setting a menu option in response to a menu item being chosen... and when you set a menu item, it generates the menu item being chosen event... do we really need to be setting the option in this case?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 37e6fce30b8b91d5d0fefce2a0ed056cf6df4ef9 available at: https://gist.github.com/d6bdc4197537ff2d081cd2025afd725e<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7510.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7510.dmg

**Comment by OmegaHeron** [comment]
Tested Windows 7 - setting aside any other concerns raised over how far this could/should go - the immediate result is updated JS fixes issue and allows device changes to properly occur.

**Pull request merged**